src/Stock.js

```diff
@@ -100,14 +144,24 @@ class Stock extends React.Component {
                                 marker: { color: 'red' },
                             }
                         ]}
-                        layout={{ width: 800, height: 600, title: `${this.state.graphMode} ${this.state.stockTicker}` }}
-                    /> :
+                        layout={{ width: 800, height: 600, title: `${this.state.graphMode}${this.state.companyName}` }} /><div className="stockInfo">
+
```

Comment:
You can probably get rid of this new line, and just have the div right over the beginning of the stock info renderings since they are relating to the same component

Reply:
Decided I would just do this change (as I already had it done while checking to see if it would help the flow of the code)